vite.config.ts

```diff
@@ -40,6 +40,7 @@ export default defineConfig({
       "pages": path.resolve(__dirname, "src/pages"),
       "styles": path.resolve(__dirname, "src/styles"),
       "typechain-types": path.resolve(__dirname, "src/typechain-types"),
+      "prebuit": path.resolve(__dirname, "src/prebuilt"),
```

Comment:
typo, maybe not needed at all?

Reply:
Thanks. It’s really not needed in most cases, but in some situations like using inside workers, these aliases are necessary. Since we already have aliases for all folders at this level, let’s have one for this as well.